src/rendering/screenspacerenderable.cpp

```diff
@@ -476,7 +476,18 @@ void ScreenSpaceRenderable::update() {}
 bool ScreenSpaceRenderable::isEnabled() const {
     return _enabled;
 }
-
+bool ScreenSpaceRenderable::isUsingRaeCoords() const
```

Comment:
Add a new line before all of these functions, and move starting bracket to the same line as the function signature